templates/templates/preview/plain_text/base.template

```diff
@@ -1,5 +1,5 @@
 
-{% include "preview/experiment_plain_text.template" %}
+{% include "preview/plain_text/experiment.template" %}
```

Comment:
Is it possible to use a relative path? If so, we could simplify down to "./experiment.template". As-is, a change in the directory structure will also cause the templates to change. That might be easy to miss...